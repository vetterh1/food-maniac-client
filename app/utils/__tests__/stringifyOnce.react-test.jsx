import stringifyOnce from '../stringifyOnce';

describe('stringifyOnce', () => {
  // Minimal props to render component
  const minParams = {
    obj: {},
    replacer: null,
    indent: 2,
  };

  const regParams = {
    obj: { one: 1, two: 2 },
    replacer: null,
    indent: 2,
  };

  it('should match its empty snapshot', () => {
    const result = stringifyOnce(minParams);
    expect(result).toMatchSnapshot();
  });

  it('should match its regular snapshot', () => {
    const result = stringifyOnce(regParams);
    expect(result).toMatchSnapshot();
  });
});
