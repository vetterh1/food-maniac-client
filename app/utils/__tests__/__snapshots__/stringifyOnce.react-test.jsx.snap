// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringifyOnce should match its empty snapshot 1`] = `"{\\"obj\\":{},\\"replacer\\":null,\\"indent\\":2}"`;

exports[`stringifyOnce should match its regular snapshot 1`] = `"{\\"obj\\":{\\"one\\":1,\\"two\\":2},\\"replacer\\":null,\\"indent\\":2}"`;
